packages/preview-middleware-client/src/adp/controllers/BaseDialog.controller.ts

```diff
@@ -61,7 +62,7 @@ export default abstract class BaseDialog extends Controller {
         const beginBtn = this.dialog.getBeginButton();
 
         const fragmentName: string = input.getValue();
-        const fragmentList: { fragmentName: string }[] = this.model.getProperty('/fragmentList');
+        const fragmentList: Fragments = this.model.getProperty('/fragmentList') as Fragments;
```

Comment:
please also get rid of this one as you did it for the others

Reply:
Fixed in [39505b0](https://github.com/SAP/open-ux-tools/pull/1918/commits/39505b01b7d9cb4ccdc43b1ab21387e06780dc7a)